### PR TITLE
Kyra styles update sep 2023

### PR DESCRIPTION
**Pull request by kyrapieterse**
# Description

### Summary by CodeRabbit

- New Feature: Added new block types "Accordion Maxi", "Pane Maxi", "Search Maxi", "Slide Maxi", and "Slider Maxi" to the WordPress editor, enhancing customization options.
- New Feature: Introduced `ColumnSizeControl`, `ParallaxControl`, `PopupSettingsControl`, and `PositionControl` components for advanced layout and design control.
- Refactor: Improved modularity and reusability of code in `shape-maxi/styles.js` by utilizing helper functions.
- Test: Added test cases for `ColorControl` component and `TextMaxi` component, improving code reliability.
- Chore: Implemented a GitHub Actions workflow for deploying web application updates.
- Style: Updated CSS styles across various components for improved user interface aesthetics.

### CODE REVIEW

File: core/blocks/class-accordion-maxi-block.php [hunk 1]
```diff
@@ -0,0 +1,513 @@
...
+                self::get_pane_content_object($props),
+            );
+
+            $response = style_processor(
+                $styles_obj,
+                $data,
+                $props,
+            );
+
+            return $response;
```
[the remark this answers is not in the document]
Comment (kyrapieterse) [reply core/blocks/class-accordion-maxi-block.php:176]: @coderabbitai give Line-by-Line Code Suggestions

File: core/blocks/class-container-maxi-block.php [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+            );
+
+            $response = style_processor(
+                $styles_obj,
+                $data,
+                $props,
+            );
+
+            return $response;
+        }
```
[the remark this answers is not in the document]
Comment (kyrapieterse) [reply core/blocks/class-container-maxi-block.php:225]: https://github.com/coderabbitai give Line-by-Line Code Suggestions

File: core/blocks/class-container-maxi-block.php [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+                    'is_hover' => true,
+                    'block_style' => $block_style
+                ]) : null,
+                'opacity' => array_key_exists('opacity-status-hover', $props) && $props['opacity-status-hover'] ? get_opacity_styles(
+                    get_group_attributes($props, 'opacity', true),
+                    true
+                ) : null,
+            ];
+
+            return $response;
```
[the remark this answers is not in the document]
Comment (kyrapieterse) [reply core/blocks/class-container-maxi-block.php:288]: @coderabbitai give Line-by-Line Code Suggestions

File: core/blocks/style-helpers/get_divider_styles.php [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+                ]),
+                'justify-content' => $obj["line-horizontal-$breakpoint"] ?? get_last_breakpoint_attribute([
+                    'target' => $prefix . "line-horizontal",
+                    'breakpoint' => $breakpoint,
+                    'attributes' => $obj,
+                    'is_hover' => $is_hover,
+                ]),
+            ];
+        }
+    }
```
[the remark this answers is not in the document]
Comment (kyrapieterse) [reply core/blocks/style-helpers/get_divider_styles.php:172]: @coderabbitai give Line-by-Line Code Suggestions

**Comment by elzadj** [comment]
@Rustam198804 the go pro button is in the wrong place

Screenshot from 2023-09-21 13-36-27

**Comment by elzadj** [comment]
Thanks @Rustam198804 

- please remove commented out code
- /src/components/video-icon-control.js is not needed
- check some css files, there is no empty line at the end (need to re-save them) https://github.com/maxi-blocks/maxi-blocks/pull/5077/files
- the text is cut off:
screenshot-wordpress local-2023 09 21-17_09_37

**Pull request closed without merging**